### DISCUSSION TITLE
User Management - Enable/Disable accounts

### DISCUSSION THREAD

**Discussion by Gijsxyz** [pterodactyl/panel#4265] [Feature Requests]
Hi!

I would like to request a new suggestion.

How about a option for admins to enable/disable accounts. This will keep the account with all the permissions set in a server but if a account has the status disabled they wont be able to view servers.
If later you want to enable the account again you change it to enable and it will just have all the permissions as before.

**Comment by devnote-dev** [comment]
Has been suggested (see #2702) however it seemed to be a bit of a controversial topic.
